### PR TITLE
Snowflake replicate action doc

### PR DESCRIPTION
**Pull request by eeliang**
### Which issues does this fix?
Create documentation for https://github.com/workato/workato/pull/22381/files

### PR submission checklist
- [x] Checked all content for grammatical errors (using Grammarly, for example)
- [x] Added new files to `docs/vuepress/sidebar.js`
- [x] Check for broken links
```bash
npm run check
```

### CODE REVIEW

File: docs/.vuepress/sidebar.js [hunk 1]
```diff
@@ -1844,6 +1844,7 @@ module.exports = [
             '/connectors/snowflake/new-row-trigger',
             'New row trigger'
           ],
+          ['/connectors/snowflake/replicate.md', 'replicate action'],
```
Comment (eeshansim) [inline docs/.vuepress/sidebar.js]: Replication action

File: docs/connectors/snowflake/replicate.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: Workato connectors - Snowflake Replicate actions
```
Comment (eeshansim) [inline docs/connectors/snowflake/replicate.md]: ...action~s~

File: docs/connectors/snowflake/replicate.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+date: 2018-02-02 06:10:00 Z
+---
+
+# Snowflake - Replicate action
+
+This action updates a selected table in your Snowflake instance. It takes a list datapill as input and inserts or updates them as rows.
+
+Workato will first detect and compare the schema of the input data and Snowflake table. If they do not match, this action will create new columns in the Snowflake table. This ensures that all the data from your input will be automatically synced in Snowflake, even if there are updates to the input schema.
+
```
Comment (eeshansim) [inline docs/connectors/snowflake/replicate.md]: extra new line

File: docs/connectors/snowflake/replicate.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+*Replicate action*
+
+## Input
+
+| Input      | Description |
+| ---------- | --- |
+| Table name | Select the table to insert your rows by typing the full table name. Snowflake is case insensitive. |
+| Unique key | Rows with unique key values found will be updated with the data provided. Rows without existing unique key values will be created with data provided. |
+| Rows       | Use a list datapill as input. |
+| Flatten columns | If you input data is formatted as an object, use this toggle to flatten the object into individual key-value pairs. |
```
Comment (eeshansim) [inline docs/connectors/snowflake/replicate.md]: Give example of flattening. Better if it's in a separate section below to illustrate better.

Example object from data source:
```json
{
  "id": 1,
  "name": "Ee Shan",
  "handles": {
    "github": "eeshansim",
    "slack": "eeshan"
  }
}
```

Unflattening:

| ID | NAME | HANDLES |
| -- | ---- | ------- |
| 1 | Ee Shan | {    \"github\": \"eeshansim\",    \"slack\": \"eeshan\"  } |

Flattened:

| ID | NAME | HANDLES_GITHUB | HANDLES_SLACK |
| -- | ---- | -------------- | ------------- |
| 1 | Ee Shan | eeshansim | eeshan |

Comment (eeliang) [reply docs/connectors/snowflake/replicate.md]: Vuepress has some issues with rendering {raw mustache}. So I used a `code block` for the json string

File: docs/.vuepress/sidebar.js [hunk 5]
```diff
@@ -1844,7 +1844,7 @@ module.exports = [
             '/connectors/snowflake/new-row-trigger',
             'New row trigger'
           ],
-          ['/connectors/snowflake/replicate.md', 'replicate action'],
+          ['/connectors/snowflake/replicate', 'Replication action'],
```
Comment (eeshansim) [inline docs/.vuepress/sidebar.js]: This should be above trigger, as the last action

File: docs/connectors/snowflake/replicate.md [hunk 6]
```diff
@@ -20,8 +19,43 @@ Workato will first detect and compare the schema of the input data and Snowflake
...
+| ID | NAME     | HANDLES_GITHUB | HANDLES_SLACK |
+| -- | -------- | -------------- | ------------- |
+| 1  | John Doe | johndoe        | john          |
 
 ## Output
 
 The output of this action is a count of the number of rows upserted.
+
+::: warning
+Snowflake has a limit of 16,384 rows for insert statements. Inputs with more than 16,384 rows will cause this job to fail.
```
Comment (eeshansim) [inline docs/connectors/snowflake/replicate.md]: Insert only or Insert and/or update?

**Pull request merged**